### DISCUSSION TITLE
Passes

### DISCUSSION THREAD

**Discussion by andriux26** [jekhokie/raspberry-noaa-v2#417] [General]
Helo  how to config 3 days Passes 
![Screenshot_20210720_180810_com android chrome](https://user-images.githubusercontent.com/5492430/126348632-a4515604-ecce-4984-aa96-63ae9b653dd3.jpg
![Screenshot_20210720_180810_com android chrome](https://user-images.githubusercontent.com/5492430/126348735-6966e322-b086-4e4c-b4f4-083395b5d640.jpg)

**Comment by dom-robinson** [comment]
See [line 75 in settings.yml](https://github.com/jekhokie/raspberry-noaa-v2/blob/1ce800de835cb18b2365fa29ce4369a943be0ddb/config/settings.yml.sample#L75) 

```# how many days to schedule passes - note this MUST be an even integer,
# and the current day counts as "1" - passes will be scheduled until midnight
# of the `days_to_schedule_passes` final day
#
# NOTE: If you want to set this value LOWER than a previously configured
#       value, you must run the schedule script manually and pass the '-x' switch
#       after re-running the ./install_and_upgrade.sh script to align the variables:
#           ./scripts/schedule.sh -x
days_to_schedule_passes: 1```
